template/lambda-sqs-worker/serverless.yml

```diff
@@ -17,15 +24,25 @@ provider:
   stackName: ${self:service}
   stage: ${env:ENVIRONMENT}
   variableSyntax: "\\${((?!AWS)[ ~:a-zA-Z0-9._@'\",\\-\\/\\(\\)]+?)}"
-  versionFunctions: false
+  versionFunctions: true
   # deploymentBucket:
   #   name: 'TODO: deploymentBucketName'
   iamRoleStatements:
+    - Action: codedeploy:PutLifecycleEventHookExecutionStatus
+      Effect: Allow
+      Resource: !Join
+        - ''
+        - - !Sub arn:aws:codedeploy:${AWS::Region}:${AWS::AccountId}:deploymentgroup
+          - ':*/'
```

Comment:
The `*` is not ideal but the logical ID for the CodeDeploy application is difficult to predict without access to plugin internals. I've PRed `serverless-plugin-canary-deployments`  to automatically infer this statement, but I don't want to block our next release on it being accepted.